compute/cloud-client/src/test/java/compute/Util.java

```diff
@@ -61,15 +61,46 @@ public abstract class Util {
   // has creation timestamp >24 hours.
   public static void cleanUpExistingInstanceTemplates(String prefixToDelete, String projectId)
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
-    for (InstanceTemplate template : listFilteredInstanceTemplates(projectId, prefixToDelete)
-        .iterateAll()) {
-      if (!template.hasCreationTimestamp()) {
-        continue;
+    try (InstanceTemplatesClient instanceTemplatesClient = InstanceTemplatesClient.create()) {
+      ListPagedResponse templates = instanceTemplatesClient.list(projectId);
+      for (InstanceTemplate instanceTemplate : templates.iterateAll()) {
+        if (!instanceTemplate.hasCreationTimestamp() || !instanceTemplate.hasId()) {
```

Comment:
As I understand it, here we are trying to check for a null value for instanceTemplate. I assume that an identifier is needed for each resource and may be better suited to eliminate null objects. Should I leave both options or remove one of them?

Reply:
Added check containPrefixToDelete() to eliminate NullPointException